mmv1/third_party/terraform/tests/resource_tags_tag_key_test.go.erb

```diff
@@ -0,0 +1,159 @@
+<% autogen_exception -%>
+package google
+
+<% unless version == "ga" -%>
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+// Tags tests cannot be run in parallel without running into Error Code 10: ABORTED
```

Comment:
Do we need any warning on this in the resource itself? Maybe that two tags can't be applied on the same resource at the same time?

Reply:
I think it should be fine for normal users of terraform - I set up a mutex lock, so a local terraform process should only ever try to create one tag at a time.